src/core/ui.js

```diff
@@ -87,8 +87,55 @@ function errWarn(msg, arr, butName, title) {
 
 function createWarnButton(butName, arr, title) {
   const buttonId = `respec-pill-${butName}`;
+  var numSteps = 20.0;
+  var boxElement
```

Comment:
So, I think all this setup should be referring to `respecUI` as a whole:
https://github.com/w3c/respec/blob/develop/src/core/ui.js#L37

The "warn button" are only yellow buttons that show warnings.